modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -107,7 +107,6 @@
 import AposModalModifiedMixin from 'Modules/@apostrophecms/modal/mixins/AposModalModifiedMixin';
 import AposDocsManagerMixin from 'Modules/@apostrophecms/modal/mixins/AposDocsManagerMixin';
 import cuid from 'cuid';
-
```

Comment:
I think you're on this, but we should get the line breaks back on this, the next and AposSchema files.